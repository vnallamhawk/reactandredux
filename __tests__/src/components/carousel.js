import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Carousel from '../../../src/components/carousel';

describe('Retail Test -- Snapshot',()=>{
    it('capturing snapshot of Carousel',()=>{
        const renderer = new ShallowRenderer();
        const result = renderer.render(<Carousel/>)
        expect(result).toMatchSnapshot();
    })
})