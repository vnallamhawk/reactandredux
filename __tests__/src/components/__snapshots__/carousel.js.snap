// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Retail Test -- Snapshot capturing snapshot of Carousel 1`] = `
<div>
  <div
    className="title"
  />
  <div
    className="carouselImage"
  >
    <Carousel
      autoPlay={false}
      axis="horizontal"
      centerMode={false}
      centerSlidePercentage={80}
      dynamicHeight={false}
      emulateTouch={false}
      infiniteLoop={false}
      interval={3000}
      onChange={[Function]}
      onClickItem={[Function]}
      onClickThumb={[Function]}
      selectedItem={0}
      showArrows={true}
      showIndicators={true}
      showStatus={true}
      showThumbs={true}
      statusFormatter={[Function]}
      stopOnHover={true}
      swipeScrollTolerance={5}
      transitionTime={350}
      useKeyboardArrows={false}
      verticalSwipe="standard"
      width="100%"
    >
      
    </Carousel>
  </div>
</div>
`;
